pvlib/bifacial.py

```diff
@@ -0,0 +1,147 @@
+plane-of-array irradiance under various conditions.
+"""
+
+import pandas as pd
+
+
+def pvfactors_timeseries(
+        solar_azimuth, solar_zenith, surface_azimuth, surface_tilt,
+        timestamps, dni, dhi, gcr, pvrow_height, pvrow_width, albedo,
+        n_pvrows=3, index_observed_pvrow=1,
+        rho_front_pvrow=0.03, rho_back_pvrow=0.05,
+        horizon_band_angle=15.,
+        run_parallel_calculations=True, n_workers_for_parallel_calcs=None):
+    """
+    Calculate front and back surface plane-of-array irradiance on
+    a fixed tilt or single-axis tracker PV array configuration, and using
+    the open-source "pvfactors" package.
+    Please refer to pvfactors online documentation for more details:
+    https://sunpower.github.io/pvfactors/
+
+    Inputs
+    ------
+    solar_azimuth: numeric (list, numpy array, or pandas Series)
+        Sun's azimuth angles using pvlib's azimuth convention (deg)
+    solar_zenith: numeric (list, numpy array, or pandas Series)
+        Sun's zenith angles (deg)
+    surface_azimuth: numeric (list, numpy array, or pandas Series)
+        Azimuth angle of the front surface of the PV modules, using pvlib's
+        convention (deg)
+    surface_tilt: numeric (list, numpy array, or pandas Series)
+        Tilt angle of the PV modules, going from 0 to 180 (deg)
+    timestamps: datetime or DatetimeIndex
+        List of simulation timestamps
+    dni: numeric (list, numpy array, or pandas Series)
+        Direct normal irradiance (W/m2)
+    dhi: numeric (list, numpy array, or pandas Series)
+        Diffuse horizontal irradiance (W/m2)
+    gcr: float
+        Ground coverage ratio of the pv array
+    pvrow_height: float
+        Height of the pv rows, measured at their center (m)
+    pvrow_width: float
+        Width of the pv rows in the considered 2D plane (m)
+    albedo: float
+        Ground albedo
+    n_pvrows: int, default 3
+        Number of PV rows to consider in the PV array
+    index_observed_pvrow: int, default 1
+        Index of the PV row whose incident irradiance will be returned. Indices
+        of PV rows go from 0 to n_pvrows-1.
+    rho_front_pvrow: float, default 0.03
+        Front surface reflectivity of PV rows
+    rho_back_pvrow: float, default 0.05
+        Back surface reflectivity of PV rows
+    horizon_band_angle: float, default 15
+        Elevation angle of the sky dome's diffuse horizon band (deg)
+    run_parallel_calculations: bool, default True
+        pvfactors is capable of using multiprocessing. Use this flag to decide
+        to run calculations in parallel (recommended) or not.
+    n_workers_for_parallel_calcs: int, default None
+        Number of workers to use in the case of parallel calculations. The
+        default value of 'None' will lead to using a value equal to the number
+        of CPU's on the machine running the model.
+
+    Returns
+    -------
+    front_poa_irradiance: numeric
```

Comment:
also for every timestamp of the simulation? Not clear to me why you've included that phrase in `df_registries` docstring but not here.

Reply:
You're right, that's not very clear. I tried to clarify it in the commits below. Let me know if that's better